apps/edxapp/vars/all/main.yml

```diff
@@ -6,7 +6,7 @@ edxapp_lms_host: "lms.{{ project_name}}.{{ domain_name }}"
 
 # -- edxapp (cms/lms)
 edxapp_image_name: "fundocker/edxapp"
-edxapp_image_tag: "hawthorn.1-1.0.6"
+edxapp_image_tag: "hawthorn.1-2.0.0"
```

Comment:
upgrade to 2.0.1?

Reply:
Yes, we need to make a new edxapp release, upgrade to 2.0.1 and remove the PATH (your previous comment).